src/udev/udev-builtin-net_id.c

```diff
@@ -353,8 +371,8 @@ static int dev_pci_slot(sd_device *dev, struct netnames *names) {
                         if (dot_or_dot_dot(dent->d_name))
                                 continue;
 
-                        r = safe_atoi(dent->d_name, &i);
-                        if (r < 0 || i <= 0)
+                        r = parse_slot(dent->d_name, &i);
+                        if (r < 0)
```

Comment:
Not a matter, but `r` is not used here, so
```
if (parse_slot(dent->d_name, &i) < 0)
        continue;
```

Reply:
Can do.